js/childrenPopUp.js

```diff
@@ -0,0 +1,75 @@
+var Toast = function(config) {
+    for(var key in config) {
+        this[key] = config[key];
+    }
+
+    this.__proto__ = PopUpNotifications;
+
+}
+
+var Modal = function(config) {
+
+    this.__proto__ = PopUpNotifications;
+
+    this.statusCreatable = false;
```

Comment:
please, do not use flags in your code, except in cases, where it is absolutely needed. This is not the case